### PR TITLE
[15.0][MIG] product_restricted_type: Migration to 15.0

### PR DESCRIPTION
**Pull request by AnnaPForgeFlow**
Standard migration to 15.0

### CODE REVIEW

**Review by JuanyDForgeflow — APPROVED** [review]
LGTM!!

**Review by GuillemCForgeFlow — APPROVED** [review]
LGTM! :+1:

**Review by AaronHForgeFlow — APPROVED** [review]
Functional review

File: product_restricted_type/models/product.py [hunk 1]
```diff
@@ -8,7 +8,7 @@ class ProductCategory(models.Model):
     _inherit = "product.category"
 
     restricted_product_type = fields.Selection(
-        string="Restricted Product Type",
+        string="Restricted Product Type ",
```
Comment (HviorForgeFlow) [inline product_restricted_type/models/product.py:11]: I think pre-commit is failing due to string has the same name as the transformed `restricted_product_type` to String. Please remove the string entry

**Review by HviorForgeFlow — CHANGES_REQUESTED** [review]

**Pull request merged**